scripts/delete_old_third_party_libs.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
We do not need any such script to remove the existing BeautifulSoup directory -- the above setup automatically creates a `beautifulsoup4-4.7.1` dir in ../oppia_tools.

Reply:
Sir i write script because when we ``` install_third_party.py ``` run then it install ``` beautifulsoup4-4.7.1 ```, but it consists of ``` beautifulsoup4-4.6.0 ``` also.
SO to delete the ``` beautifulsoup4-4.6.0 ``` i write this script.

Reply:
You need not delete the existing directory if it exists in the first place.

Reply:
but why sir, is it good to have both ``` beautifulsoup4-4.7.1 ```  ``` beautifulsoup4-4.6.0 ``` in ``` /third_party ``` folder ?

Reply:
A couple of points here:
1. Having both the dirs does not make much difference as long as we specify the one we want to use.
2. If the dev wants to clean up the setup files, he/she can run clean.sh script.

Reply:
ok sir

Reply:
ok sir, Done.